### ISSUE TITLE
Seamless Waypoint

### ISSUE THREAD

**Issue by DavidxRussell** [dawson-vilamaa/BeaconWaypoints#20]
I would love Shift+Right-Click a Beacon to create a Waypoint! Being able to create a Waypoint with no commands would make this feel so intuitive with the rest of the game.

**Comment by Pandemonious** [comment]
yeah. The /waypoint command being required is pretty immersion-breaking.

by the way, I was toying with placeholderAPI and commandpanels. It should be possible to have an option appear in a menu if placeholder api takes the player's block coordinates, subtracts 1 from the Y coordinate, and then checks if the block in that position is a beacon. I'm going to mess with this some more and report back to help you out @DavidxRussell
